src/document/document.wrapper.controller.js

```diff
@@ -34,4 +34,25 @@ angular.module('engine.document')
                 $location.search({step: newVal || 0})
             }
         });
-    });
\ No newline at end of file
+    })
+    .directive('fixedOnScroll',['$window',function ( $window ) {
+    var $win = angular.element($window);
+    return {
+        restrict: 'A',
+        link: function (scope, element, attrs) {
+            var topClass = attrs.fixedOnScroll;
+            var offsetTop =  element[0].getBoundingClientRect().top;
+
+            $win.on("scroll", function () {
+
+                if ($window.pageYOffset >= offsetTop) {
```

Comment:
This doesn't really work as intended, I guess. Two problems:
1. When I start scrolling, the class is added, but the sidebar skips up when it is added. Instead, this should be fluent. See affix component on https://getbootstrap.com/docs/3.3/javascript/#affix and the behaviour of the right menu there.
2. When I open the page with a narrow window and widen it later, the sidebar expands. Then, if I scroll, the class isn't removed. So that sidebar floats over the top menu even in the top-most scroll position.